Multisig/UI/Transaction/Execution/ReviewExecutionViewController.swift

```diff
@@ -583,10 +583,17 @@ class ReviewExecutionViewController: ContainerViewController {
                 trackedKeyType = "ledger_nano_x"
             }
 
-            successVC.trackingParams = [
+            let trackingParams = [
                 "keyType": trackedKeyType,
                 "chainId": self.controller.chainId
             ]
+            
+            if key.keyType == .walletConnect {
+                let walletName = key.wallet?.name ?? "Unknown"
```

Comment:
this will fail for the connections via qr code, please see my other pr - we would need to use the connection's remote peer name to get the name in case we don't have 'wallet'